Development/src/pages/receivers/ReceiversShow.js

```diff
@@ -424,14 +446,18 @@ const ConnectionManagementTab = ({
                                 selected={get(receiverData, 'id') === item.id}
                             >
                                 <TableCell component="th" scope="row">
-                                    <ShowButton
-                                        style={{
-                                            textTransform: 'none',
-                                        }}
-                                        basePath="/senders"
-                                        record={item}
-                                        label={item.label}
-                                    />
+                                    <Link
+                                        to={`${linkToRecord(
```

Comment:
Would a `ReferenceField` work here?

Reply:
Difference is ReferenceField will make another network request although it would look more simple.

Reply:
Ah, yes, that makes sense, and is a good reason. Please add that as a comment here.